docs/OnTrack/Deployment/Enhanced Authentication/testing-strategy-enhance-authentication.md

```diff
@@ -0,0 +1,59 @@
+# Testing Strategy for Enhance Authentication
+
+[Introduction](#introduction)
+
+[Overview of Deliverables to be Tested](#overview-of-deliverables-to-be-tested)
+
+[References](#references)
```

Comment:
I would put the references as the bottom of the document - almost as an appendix. It isn't 100% required knowledge for the testing strategy but it is useful to have.

Reply:
You'll have to blame me for this ordering, @talzid - it follows the order that comes from our handbook testing strategy template (based on a format from _Programming Foundations: Software Testing/QA from LinkedIn Learning_).

Reply:
Nevermind my comment then. If Kat says it is, then that it shall be :smile: I trust the method and research